src/components/atoms/SideBarButton/SideBarButton.tsx

```diff
@@ -0,0 +1,16 @@
+import { SideBarButtonProps } from './ButtonInterface';
+
+const SideBarButton = ({ title, component: Icon }: SideBarButtonProps) => {
+  return (
+    <div className="h-[25px] w-[170px]  flex items-center justify-between ">
```

Comment:
no manual classes please. move that to config OR use a class which is providing a similar config